### PR TITLE
LPD-26188 Created Templates Blocks and Company

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/marketplace-solutions-blocks/ddm-template.ftl [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+<#assign channel=restClient.get("/headless-commerce-delivery-catalog/v1.0/channels?accountId=-1&filter=name eq 'Marketplace Channel' and siteGroupId eq '${scopeGroupId}'") />
+
+<#if channel?has_content>
+	<#assign channelId = channel.items[0].id />
+</#if>
+
+<#assign
+	productId = CPDefinition_cProductId.getData()
+	product = restClient.get("/headless-commerce-delivery-catalog/v1.0/channels/"+ channelId +"/products/"+ productId +"?accountId=-1&images.accountId=-1&nestedFields=productSpecifications,categories,images")
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/marketplace-solutions-blocks/ddm-template.ftl:68]:
```suggestion
	product = restClient.get("/headless-commerce-delivery-catalog/v1.0/channels/"+ channelId +"/products/"+ productId +"?accountId=-1&images.accountId=-1&nestedFields=categories,images,productSpecifications")
```

**Review by joanagrego — CHANGES_REQUESTED** [review]

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:sf

**Pull request closed without merging**